src/wp-admin/includes/media.php

```diff
@@ -3791,7 +3791,7 @@ function wp_get_media_creation_timestamp( $metadata ) {
 		case 'matroska':
 		case 'webm':
 			if ( isset( $metadata['matroska']['comments']['creation_time'][0] ) ) {
-				$creation_date = strtotime( $metadata['matroska']['comments']['creation_time'][0] );
+				$creation_date = ( new DateTimeImmutable( $metadata['matroska']['comments']['creation_time']['0'] ) )->getTimestamp();
```

Comment:
Why `['0']` instead of `[0]`

If this is required imho this should be documented, otherwise future developers will remove this, as they think it's an error. And if it's by accident, it should be reverted to `[0]`.